src/components/ActionsPanel/components/TweetButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useContext } from 'react';
+
+import { StoreContext } from '@services';
+import { translateRaw } from '@translations';
+import { ExtendedUserAction } from '@types';
+import { generateTweet } from '@utils';
+
+import { ActionButton } from './ActionButton';
+
+export interface TweetButtonProps {
+  userAction: ExtendedUserAction;
+}
+
+export const TweetButton = ({ userAction }: TweetButtonProps) => {
+  const { memberships } = useContext(StoreContext);
+
+  const to = generateTweet(
+    translateRaw('MYC_WINTER_BONUS_TWEET', {
```

Comment:
Can we add this as a prop so it is a more reusable component? If not, no worries.

Reply:
I can't actually - Otherwise I would've used the `ActionButton` but I need a wrapper to get the Membership address -